lab/lab1/part1-array-access.html

```diff
@@ -79,6 +79,14 @@
     L.marker([0, 0]).addTo(map);
     L.marker([0, 0]).addTo(map);
 
+var header = ['lat','lng','label'];
+var TempArr = [[39.952238, -75.201702,'Top1'],[39.952927, -75.217711,'Top2'],[39.944143, -75.174881,'Top3']];
+TempArr.unshift(header);
+for (i=0; i < TempArr.length; i++){console.log(TempArr[i])};
+var makeAMarker = (obj) => {return L.marker([obj[0], obj[1]]).bindPopup(obj[2])}
+for (var i=1; i <= TempArr.length; i++){ makeAMarker(TempArr[i]).addTo(map) }
```

Comment:
`<=` is giving this error: 

![image](https://user-images.githubusercontent.com/8103418/52543915-79d8f600-2d7b-11e9-9b53-844406ed2bc4.png)


Try without the `=` it resolves the issue.

Reply:
Changed. This solved the propblem!